backend/lib/til_web/router.ex

```diff
@@ -3,9 +3,16 @@ defmodule TilWeb.Router do
 
   pipeline :api do
     plug :accepts, ["json"]
+    plug TilWeb.Plug.AuthAccessPipeline
+  end
+
+  scope "/auth", TilWeb do
+    get "/:provider", AuthController, :request
```

Comment:
does `ueberauth` validates provider? since we are only using `google` will it allow for others (it should not)?

Reply:
It validates provider, there is nothing happen when enter other providers i guess cause we implemented only google strategy.

Reply:
if there is a validation then it's fine